CalcualtorLogic.js

```diff
@@ -28,6 +28,9 @@ class Calculator {
     constructor(selector) {
         this.hostElem = document.querySelector(selector);
     };
-    render() {};
+    render() {
+        const wrapperDiv = document.createElement('div');
+        wrapperDiv.classList = 'wrapper';
```

Comment:
This is incorrect `classlist` is an object you can not assign value like this
the `classlist` has `add` method which can be used to add a class to the element.